fcrepo-clients/src/main/java/edu/unc/lib/dl/fcrepo4/FcrepoClientFactory.java

```diff
@@ -83,6 +103,7 @@ public FcrepoClient makeAuthenticatedClient(String host, String user, String pas
                 .credentials(user, password)
                 .authScope(host)
                 .throwExceptionOnFailure()
+                .hostHeader(hostHeader)
```

Comment:
what happens here when hostHeader is null?

Reply:
the client built by the factory won't add the host header